lib/dry/system/component.rb

```diff
@@ -7,7 +7,7 @@
 require "dry/system/loader"
 require "dry/system/errors"
 require "dry/system/constants"
-require "dry/system/magic_comments_parser"
+require_relative "identifier"
```

Comment:
@timriley out of curiosity - why `require_relative` here?

Reply:
@solnic It's because I'd like to move most of the requires for "neighbouring" files to require_relative (it's slightly faster and it also is a clear signal that a file is coming from nearby), but I didn't want to touch lines that were not part of this overall change. I'll come back for those in a follow-up PR sometime soon.

Reply:
@timriley ahh, great - do you happen to know how faster this is? We could totally change it everywhere in a batch-change to speed things up (across all the repos).